spec/system/candidate_completing_an_application_spec.rb

```diff
@@ -1,29 +1,15 @@
 require 'rails_helper'
```

Comment:
Where'd the spec go?

Reply:
The file is there, I am just testing **only** the final submit button.
I was not sure of adding the spec containing all the steps about filling all the form, it sounds redundant and very slow,  let's discuss how we can test the full submission of the application in a more efficient way.